classes/gui/ScatterView2.sc

```diff
@@ -1,12 +1,54 @@
-// 2008 by till bovermann
-//   bielefeld university
+/*********************************************************************
+(C) 2008 Till Bovermann / Bielefeld University
 
+and later...
+
+(C) 2018 Jonathan Reus / IEM Graz
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+    For more details see: https://www.gnu.org/licenses/
+**********************************************************************/
+
+
+
+
+
+
+
+/*
```

Comment:
is this the dopcumentation? is this actually rendered? if not, could it go into a standard scdoc file?

Reply:
not actually rendered, so it could also go into the help file